ReSharper.FSharp/Directory.Build.props

```diff
@@ -12,6 +12,7 @@
     <FSharpCoreVersion>5.0.0</FSharpCoreVersion>
     <FantomasVersion>2020.3.0</FantomasVersion>
     <ValueTupleVersion>4.5.0</ValueTupleVersion>
+    <CSharpLanguageVersion>8</CSharpLanguageVersion>
```

Comment:
Please don't put it to the group that specifies package versions, it becomes a mess.